### PR TITLE
[ENH] Might need info in camt import from PmtInfId element.

### PR DESCRIPTION
**Pull request by NL66278**
Needed to import some transactions, with entry details like these:

```
<NtryDtls>
  <Btch>
    <PmtInfId>201511-20151113103109              </PmtInfId>
  </Btch>
  <TxDtls><BkTxCd>
  <Prtry>
  <Cd>584</Cd>
  </Prtry>
  </BkTxCd>
  <Purp>
    <Cd>SALA</Cd>
  </Purp>
  </TxDtls>
</NtryDtls>
```

### CODE REVIEW

File: account_bank_statement_import_camt/camt.py [hunk 1]
```diff
@@ -137,6 +137,13 @@ def parse_transaction(self, ns, node, transaction):
         if not transaction.message:
             self.add_value_from_node(
                 ns, node, './ns:AddtlNtryInf', transaction, 'message')
+        if not transaction.eref:
+            self.add_value_from_node(
+                ns, node, [
+                    './ns:NtryDtls/ns:Btch/ns:PmtInfId',
+                ],
+                transaction, 'eref'
+        )
```
Comment (gurneyalex) [inline account_bank_statement_import_camt/camt.py]: add 4 spaces to this line to get a green build :smile_cat:

**Comment by NL66278** [comment]
@gurneyalex Thanks Alex, I have corrected the indentation.

**Comment by hbrunn** [comment]
:+1:

**Comment by sbidoul** [comment]
:+1:

**Pull request merged**